### PR TITLE
Feature/dfe 1021 api call admin dashboard publications part 3

### PR DESCRIPTION
**Pull request by richardpopplehive**

### CODE REVIEW

File: src/GovUk.Education.ExploreEducationStatistics.Content.Model.Test/ReleaseTests.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Test
+{
+    public class ReleaseTests
+    {
+        [Fact]
+        public void Release_Not_Live()
+        {
+            {
```
Comment (jamescutts) [inline src/GovUk.Education.ExploreEducationStatistics.Content.Model.Test/ReleaseTests.cs]: Shouldn't need these extra { }.

Might be worth splitting this into two seperate tests

Comment (richardpopplehive) [reply src/GovUk.Education.ExploreEducationStatistics.Content.Model.Test/ReleaseTests.cs]: Split into two test methods

File: src/GovUk.Education.ExploreEducationStatistics.Content.Model.Test/ReleaseTests.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+using System;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Test
```
Comment (jamescutts) [inline src/GovUk.Education.ExploreEducationStatistics.Content.Model.Test/ReleaseTests.cs]: Can we rename the project and namespace from `Test` to `Tests` this keeps it consistient and also simplies the pipline with runs tests for all project that end in `Tests` or `tests`

Comment (richardpopplehive) [reply src/GovUk.Education.ExploreEducationStatistics.Content.Model.Test/ReleaseTests.cs]: Done

**Pull request merged**